artifacts/openapi.yaml

```diff
@@ -183,6 +183,26 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /control/protocols:
+    post:
+      tags:
+      - Control
+      operationId: set_protocol_state
+      description: |-
+        Sets all configured protocols to intended state
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Protocol.State'
+      responses:
```

Comment:
Just a query , not a comment. In the generated .proto all the return types are of a single type called Response inside which there is an array of strings ( error strings or warning strings ) . What is the signficance of the Success/BadRequest and InternalServerError below. Is this hidden from user in the reply via .proto format and user sees these same responses as Response messages ?

Reply:
- `Success`: should contain warnings and any other useful data structures
- `BadRequest`: should at least contain a list of errors for validation errors (e.g. `pause` is not a correct state, `eth1` is not a valid device name, etc.)
- `InternalServerError`: should at least cotain a list of errors for actual errors internal to application (e.g. devices are all valid but starting protocols on some failed)